nwalsdev/templates/account/login.html

```diff
@@ -1,6 +1,6 @@
 {% extends "site_base.html" %}
 
-{% load account_tags %}
```

Comment:
Why is this being removed? If we don't need it, is there any reason to leave the empty code brackets?